seeder.js

```diff
@@ -23,23 +23,27 @@ mongoose.connect(url, (err) => {
       db.dropDatabase();
       const reviews = [];
       for (let i = 0; i < 100; i += 1) {
+        const propId = i;
         const property = faker.company.companyName();
         const reviewCount = Math.floor(Math.random() * 50) + 25;
         // console.log(reviewCount);
         // console.log(i);
         for (let j = 0; j < reviewCount; j += 1) {
+          const revId = j;
           const review = new Review ({
+            propertyId: propId,
+            reviewId: revId,
             propertyName: property,
             username: faker.name.firstName(),
             avatar: faker.internet.avatar(),
             review: faker.lorem.paragraph(),
             dayPosted: faker.date.past(),
             rating: {
-              cleanliness: faker.random.number({ min: 0, max: 5 }),
-              communication: faker.random.number({ min: 0, max: 5 }),
-              accuracy: faker.random.number({ min: 0, max: 5 }),
-              checkIn: faker.random.number({ min: 0, max: 5 }),
-              location: faker.random.number({ min: 0, max: 5 }),
+              cleanliness: faker.random.number({ min: 1, max: 5 }),
+              communication: faker.random.number({ min: 1, max: 5 }),
+              accuracy: faker.random.number({ min: 1, max: 5 }),
+              checkIn: faker.random.number({ min: 1, max: 5 }),
+              location: faker.random.number({ min: 1, max: 5 }),
```

Comment:
What's the advantage of using 1 instead of 0 as the minimum?

Reply:
I realized that a lot of the time the minimum rating you can give is 1. I was assuming it was the same for airbnb